app/js/skrollr.js

```diff
@@ -29,17 +29,30 @@ angular.module("sn.skrollr", [])
          */
         function($window, $document, $rootScope) {
 
-            return {
+            var s,
+                serviceMethods,
+                init = 0;
+
+            serviceMethods = {
                 init: function() {
 
                     $document.ready(function () {
                         $rootScope.$apply(function() {
```

Comment:
You need to check if theres already a digest cycle happening before calling `$apply` else you'll get an error and your code will not run, you should wrap your code like this: 

https://gist.github.com/eopa/a4c3cc7ac1f9c772dc10